src/hub.js

```diff
@@ -248,7 +248,12 @@ const onReady = async () => {
     document.body.removeEventListener("touchend", requestFullscreen);
   };
 
+  let hasEnteredScene = false;
+
   const enterScene = async (mediaStream, enterInVR, hubId) => {
+    if (hasEnteredScene) return;
+    hasEnteredScene = true;
```

Comment:
What case is this handling? It's not obvious to me how it would happen that `enterScene` is called twice.

Reply:
you're right I think this is leftover from earlier changes